LL-1267 Extract non-translatables terms for English, styles + backport

juan-cortes opened the pull request:
This pr should address the inconsistencies in translating non-translatable strings such as dongle screen texts that would deem the instructions useless since they wouldn't match what the user is seeing. Also brings **bold text** to those non-translatable strings on the instructions for nanox, they weren't highlighted.

<img width="1385" alt="Screenshot 2019-04-04 at 18 07 15" src="https://user-images.githubusercontent.com/4631227/55572222-702e9780-5707-11e9-8123-65ed066c32c4.png">


> There is at least one case in Chinese where I was unable to backport the translation, I'm assuming crowdin will do somethinga bout it when the translation changes

gre commented:
some ideas we could improve:

- we need to remove from translation the account name things. the `(legacy) (unsplit)` things.
- for errors, each time we have `title: "{{message}}"` , I suggest we remove BUT in the TranslatedError we will fallback on `error.message` by default if no translation is available. tell me wdyt about this, just a thought, but i hate seeing these `{{message}}` useless things :D

juan-cortes commented:
I agree. It's a first step, there are several useless literals with only a placeholder for something else, like the repair device options right now. They shouldn't have to go through i18n, I'll take a look

juan-cortes commented:
This is now updated, I had some issues with the merge with Korean, same as I did with Chinese. If both cases I replaced the wording back to English version since I was unable to locate the non-translatable wording in the string.

gre commented:
@juan-cortes conflict again, sorry 😢 

I will set this PR as important because we want it merged soon so crowdin don't collide each time.

gre commented:
@Arnaud97234 i'm not sure this PR is the point for these discussion but more to be discussed directly on Crowdin or internally

gre commented on static/i18n/en/app.json:
```diff
@@ -526,9 +526,9 @@
-      "followDeviceRepair": "Follow device repair instructions"
+      "generic": "{{repairDeviceBootloader}}",
+      "mcuNotGenuine": "{{repairDeviceMcuNotGenuine}}",
+      "followDeviceRepair": "{{repairDeviceFollow}}"
```
these 3 wordings should be removed from translation since there is no value to add them.

juan-cortes replied:
I see

juan-cortes commented:
This should be ready to merge now, every time we get new translations I have to manually update the language files and it's time-consuming and error-prone -.-
